CHANGELOG.md

```diff
@@ -132,8 +132,8 @@ upgrade your Kong cluster.
 ##### Configuration
 
 - :warning: The `server_tokens` and `latency_tokens` configuration properties
-  have been removed. Instead, a new `headers` configuration properties replaces
-  them and allows for a more granular settings of injected headers (e.g.
+  have been removed. Instead, a new `headers` configuration properties replace
+  them and allow for a more granular setting of injected headers (e.g.
```

Comment:
ditto for `allows`?

About the other fix here: I believe `for more granular settings of injected headers` was what we were aiming at, since this option allows for several combinations of settings (hence the granularity).